common/app/layout/Front.scala

```diff
@@ -259,7 +259,6 @@ case class FaciaContainer(
 
   def isStoryPackage: Boolean = container match {
     case Dynamic(DynamicPackage) => true
-    case Dynamic(DynamicElection) => true // #election2017
     case _ => false
   }
```

Comment:
Maybe don't change it now, especially if already tested without.  It does look like something that might be added to again anyway.